app/javascript/packs/home/index/button_click.js

```diff
@@ -25,17 +25,18 @@ $(document).on("click", ".show-comment-btn", function() {
     }
   });
 });
-// $(function() {
-//   $("#my_form").on("submit", function(event) {
-//     $.ajax({
-//       type: "POST",
-//       url: this.action,
-//       data: $(this).serialize(),
-//       success: function(response) {
-//         //update the DOM
-//       }
-//     });
-//     event.preventDefault();
+$(document).on("click", ".like", function() {
+  likeButton = $(this).attr("id");
+  console.log(likeButton);
+  $(this).toggleClass("color-blue ");
+  // likeButton.prop("disabled", true);
+});
+
+// $(document).ready(function() {

```

Comment:
Remove code which is not functional instead of commenting it.

_**P.S You can make a code snippets file in which you can save such reusable code components**_